docs/reference/core/runtime.md

```diff
@@ -82,20 +83,243 @@ We can easily access the default `Runtime` to run an effect:
 ```scala mdoc:compile-only
 object MainApp extends scala.App {
   val myAppLogic = ZIO.succeed(???)
+
   val runtime = Runtime.default
+
   Unsafe.unsafe { implicit unsafe =>
     runtime.unsafe.run(myAppLogic).getOrThrowFiberFailure()
   }
 }
 ```
 
-## Custom Runtime
+## Top-level And Locally Scoped Runtimes
+
+In ZIO, we have two types of runtimes:
+
+- **Top-level runtime** is the one that is used to run the entire ZIO application from the very beginning. There is only one top-level runtime when running a ZIO application.
```

Comment:
Another common use case here is creating a top level runtime in a mixed application. For example, if we are using an HTTP library that does not have direct support for ZIO we may need to use `Runtime.unsafe.run` in the implementations of each of our routes.